src/routes/about/+page.svelte

```diff
@@ -1,41 +1,328 @@
+				<li>
+					<hr />
+					<div class="timeline-middle">
+						<svg
+							xmlns="http://www.w3.org/2000/svg"
+							viewBox="0 0 20 20"
+							fill="currentColor"
+							class="w-5 h-5"
+							><path
+								fill-rule="evenodd"
+								d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.857-9.809a.75.75 0 00-1.214-.882l-3.483 4.79-1.88-1.88a.75.75 0 10-1.06 1.061l2.5 2.5a.75.75 0 001.137-.089l4-5.5z"
+								clip-rule="evenodd"
+							/></svg
+						>
+					</div>
+					<div class="timeline-end timeline-box">Example</div>
+					<hr />
+				</li>
+				<li>
+					<hr />
+					<div class="timeline-start timeline-box">Example</div>
+					<div class="timeline-middle">
+						<svg
+							xmlns="http://www.w3.org/2000/svg"
+							viewBox="0 0 20 20"
+							fill="currentColor"
+							class="w-5 h-5"
+							><path
+								fill-rule="evenodd"
+								d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.857-9.809a.75.75 0 00-1.214-.882l-3.483 4.79-1.88-1.88a.75.75 0 10-1.06 1.061l2.5 2.5a.75.75 0 001.137-.089l4-5.5z"
+								clip-rule="evenodd"
+							/></svg
+						>
+					</div>
+				</li>
+			</ul>
+		</div>
+
+		<p class="font-semibold text-xl mt-10 mb-5 ml-5">Join Us On Discord!</p>
+
+		<div id="contact-information ">
+			<a href="https://discord.gg/2ytj4pkUPY">
+				<button
+					class="relative inline-flex items-center justify-center w-16 p-0.5 ml-5 mb-20 me-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-cyan-500 to-blue-500 group-hover:from-cyan-500 group-hover:to-blue-500 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-cyan-200 dark:focus:ring-cyan-800"
+				>
+					<img src="/discord1.svg" class="w-12" /></button
+				>
+			</a>
+		</div>
+
+		<div class="contact-form">
+			<p class="font-semibold text-xl mt-0 mb-5 text-white pt-10">Contact Form</p>
+			<input type="text" name="first" placeholder="First Name" autocomplete="off" required />
+			<input type="text" name="last" placeholder="Last Name" autocomplete="off" required />
+			<input type="email" name="Email" placeholder="Email Address" autocomplete="off" required />
+			<textarea
+				rows="5"
+				cols="50"
+				name="message"
+				placeholder="Enter text"
+				autocomplete="off"
+				required
+			/>
+			<button type="submit" id="submit-button">Send Message</button>
+		</div>
 
-	<div id="legal-information">
-		Provide necessary legal disclaimers, privacy policies, or terms of service to ensure compliance
-		with relevant regulations and protect user rights.
+		<div id="legal-information">
+			<p class="font-semibold text-xl mt-0 mb-5 ml-5 text-white pt-10">Legal</p>
+
+			<p class="text-sm mb-5 ml-5 mr-5">
+				Living Dex Tracker is a fan-made website dedicated to providing information about Pokemon
+				creatures. We do not claim ownership of any Pokemon characters, images, or other content
+				featured on this website.
+				<br /><br />We do not generate revenue from the use of Pokemon content. Any advertisements
```

Comment:
Please can this be a second paragraph tag, rather than having two `br` tags to split the paragraphs.